src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/SignIn.razor.cs

```diff
@@ -60,17 +60,17 @@ private async Task DoSignIn()
             return;
         }
 
-        IsLoading = true;
-        IsSignInButtonEnabled = false;
-        SignInMessage = null;
-
         if (ValidateSignIn() is false)
         {
             IsLoading = false;
```

Comment:
The reason behind doing this change, was changing followings:

```cs
if (ValidateSignUp() is false)
        {
            IsLoading = false; // must gets removed
            IsSignUpButtonEnabled = true; // must gets removed
            return;
        }
```

to followings:

```cs
if (ValidateSignUp() is false)
        {
            return;
        }
```

Reply:
You mean something like this?:

```cs
if (ValidateSignIn() is false)
{
    return;
}

IsLoading = true;
IsSignInButtonEnabled = false;
SignInMessage = null;
```
Or this?:
```cs
IsLoading = true;
IsSignInButtonEnabled = false;
SignInMessage = null;

if (ValidateSignIn() is false)
{
    return;
}
```

And should I apply them to both files ([SignUp.razor.cs](https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/SignUp.razor.cs) and [SignIn.razor.cs](https://github.com/bitfoundation/bitframework/blob/develop/src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/SignIn.razor.cs)) ?

Reply:
I'd appreciate first approach in both files